### ISSUE TITLE
Clicking on deviantart feed doesn't link to user's dA profile

### ISSUE THREAD

**Issue by spriteclad** [kickscondor/fraidycat#176]
**Before you start.**

* [x] I have checked the [Common Problems](https://github.com/kickscondor/fraidycat/wiki/Common-Problems) page.
* [x] I have searched the issue tracker for a bug that matches the one I want to file, without success.

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Firefox
 - Version: 79
 - URL you are attempting to follow: https://www.deviantart.com/thevoidful

**Describe the bug**
Basically when I add said URL to Fraidycat it gets feed items just fine, but whenever I click on the feed name in Fraidycat, it takes me to https://www.deviantart.com/newest/?q=gallery%3Athevoidful which redirects to https://www.deviantart.com/search/deviations?q=gallery%3Athevoidful&order=most-recent which doesn't display that person's deviations at all when clicked. It just gives me a blank search request. Any idea why this is?

**Comment by medicinecal** [comment]
Ok great - this change has been made. You may have to wait a few hours for the change to get picked up. Thanks for checking in!

**Comment by spriteclad** [comment]
Thank you so much for fixing this! I really appreciate it! :D

**Comment by medicinecal** [comment]
Ok great comment - I've updated it to use /gallery like you say. Thank you!
